scripts/mkcloud

```diff
@@ -55,6 +55,7 @@ fi
 # causes of this is violation of the common shell coding standard which
 # uses uppercase for environment variables and constants, and lowercase
 # for local variables.
+: ${mkcloud_lib_dir:=$(dirname $0)/lib}
```

Comment:
Maybe even more flexible:
```
+: ${mkcloud_dir:=$(dirname $0)}
+: mkcloud_lib_dir=${mkcloud_dir}/lib
```
With this we could also fetch other scripts like qa_crowbarsetup from a different location (and drop the current special variable for it).